apps/api/src/app/subscribers/e2e/update-global-preference.e2e.ts

```diff
@@ -27,7 +27,7 @@ describe('Update Subscribers global preferences - /subscribers/:subscriberId/pre
     };
 
     try {
-      const firstResponse = await novuClient.subscribers.preferences.updateGlobal(
```

Comment:
question: why do we have different data structure patterns?
above we have listLegacy and retrieveByLevelLegacy with suffix legacy but here we have nested object of legacy.

Reply:
@BiswaViraj ☝️

Reply:
I am not sure about that, it was already present before, after rebase the tests were failing so i just updated the e2e tests to use the available methods